Portfolio2.css

```diff
@@ -0,0 +1,222 @@
+* {
+	margin:0;
+	padding: 0;
+	
+}
+
+body {
+	background-image: url("https://wallpaperaccess.com/full/1128684.jpg");
+	background-size: cover;
+	
+	
+ }
+
+header {
+	display: inline-flex; 
+	position: fixed; 
+	margin-left: 1200px;
+	position:fixed;
+	margin-top:-180px;
+}
+
+.menue ul li {
+	display: inline-flex;
```

Comment:
This should be inline-block, not inline-flex.

It will look the same, but since these li do not contain flex items, there is no point in enabling flex